tstest/natlab/vnet/vnet_test.go

```diff
@@ -194,7 +221,67 @@ func mkAllNodesPing(srcMAC MAC, srcIP netip.Addr) []byte {
 		TypeCode: layers.CreateICMPv6TypeCode(layers.ICMPv6TypeEchoRequest, 0),
 	}
 	icmp.SetNetworkLayerForChecksum(ip)
-	return mkEth(macAllNodes, srcMAC, layers.EthernetTypeIPv6, mkPacket(ip, icmp))
+	return mkEth(macAllNodes, srcMAC, ethType6, mkPacket(ip, icmp))
+}
+
+// mkDNSReq makes a DNS request to "control.tailscale" using the source IPs as
+// defined in this test file.
+//
+// ipVer must be 4 or 6. If 4, it makes a A record request. If 6, it makes a
```

Comment:
Makes an?

Reply:
done (on first one. for second, I read it as "a quad A", so American article rules say "a" not "an"; British says how it's written IIRC, as opposed to how it sounds when read)